src/test/ui/generic-associated-types/issue-88287.stderr

```diff
@@ -0,0 +1,27 @@
+error[E0277]: the size for values of type `A` cannot be known at compilation time
+  --> $DIR/issue-88287.rs:35:9
+   |
+LL | type SearchFutureTy<'f, A, B: 'f>
+   |                         - this type parameter needs to be `std::marker::Sized`
+...
+LL |         async move { todo!() }
+   |         ^^^^^^^^^^^^^^^^^^^^^^ doesn't have a size known at compile-time
+   |
```

Comment:
Don't know if it's just me, but I'm having a difficult time actually figuring out what the problem here is...

Reply:
yea, but that's not a generator issue, closures would have the same issue.